liana/resource/_select_resource.py

```diff
@@ -4,7 +4,7 @@
 from pandas import DataFrame
 
 
-def select_resource(resource_name: str = 'consensus') -> DataFrame:
+def select_resource(resource_name: str) -> DataFrame:
```

Comment:
I assume this is prelim, but there should be only one file included to liana. Also, leave consensus as default. Only change it so that metalinks calls the 'metalinksdb' resource by default. Defining what metalinksdb is should be priority (together with having a method that we believe in more than the mean/product).

Reply:
resource pulling has to be reworked in general right ?